org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF

```diff
@@ -27,7 +27,8 @@ Require-Bundle: org.eclipse.xtext.junit4,
  org.eclipse.xtext.builder,
  org.hamcrest.core;bundle-version="1.1.0",
  org.eclipse.ui.ide;bundle-version="3.6.0",
- org.eclipse.xtext.common.types.shared
+ org.eclipse.xtext.common.types.shared,

```

Comment:
Unfortunately there are still some tests that use the old AbstractXtextTests infrastructure, that's why I couldn't remove references to junit4. I don't think we want to port the old tests, or at least, that was my understanding ;)